.circleci/config.yml

```diff
@@ -1,24 +1,28 @@
 version: 2
 
 jobs:
-  build:
-    docker:
-      - image: circleci/golang:1.11.4
-    working_directory: /go/src/github.com/cloudwan/gohan
-    environment:
-        GO111MODULE: 'on'
-    steps:
-      - checkout
-      - run: go version
-      - run: make deps gen lint build
```

Comment:
Why is this job removed? I know that target `test` depends on `build` and that depends on `deps`, so we aren't losing those steps. But we don't run linters anymore on CI with this change.